runbotics/runbotics-desktop/src/action/microsoft/excel/excel.service.ts

```diff
@@ -197,6 +197,21 @@ export class ExcelService {
                 this.getSessionHeader(session),
             );
     }
+    /**
+     * @see https://learn.microsoft.com/en-us/graph/api/worksheetcollection-add?view=graph-rest-1.0&tabs=javascript
+     */
+    async createWorksheet(session: ExcelSession, worksheetName: Worksheet['name']) {
+        await this.microsoftGraphService
+            .post(
+                this.createWorkbookUrl(session, '/worksheets/add'),
+                {
+                    ...this.getSessionHeader(session),
+                    name: worksheetName
+                }
+            );
+
+        return worksheetName;
```

Comment:
If this is possible, it would be better to return worksheet name retrieved from the response object.